integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -1139,9 +1136,25 @@ object BloopDefaults {
     BloopKeys.bloopGenerate
       .all(filter)
       .map(_.toSet)
+      .map(_.foldLeft(Set.empty[Option[File]] -> Set.empty[Incomplete]) {
+        case ((succ, fail), res) =>
+          res match {
+            case Inc(th) => succ -> (fail + th)
```

Comment:
```suggestion
            case Inc(incomplete) => succ -> (fail + incomplete)
```
or failure or something, I'm not 100% sure. But `th` isn't very clear to me.

Reply:
done